src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx

```diff
@@ -57,11 +56,15 @@ export class VerifyCollectorStep extends PureComponent<Props> {
             configID={telegrafConfigID}
           />
         </DapperScrollbars>
-        <OnboardingButtons
-          onClickBack={onDecrementCurrentStepIndex}
-          nextButtonText="Finish"
-          className="data-loading--button-container"
-        />
```

Comment:
We didn't need the collection of buttons anymore so i just removed this and added a simple button to the footer.